app/views/partials/_nav.html.haml

```diff
@@ -2,6 +2,8 @@
 -if user_signed_in?
     %p Hello, #{current_user.email}
     =link_to "Logout", destroy_user_session_path, method: :delete
+    =link_to 'Edit profile', edit_user_registration_path
```

Comment:
Is it not better to have the "Edit profile" link on the "View Profile" page ?

Reply:
done